specs-go/config.go

```diff
@@ -692,6 +692,7 @@ const (
 	ArchPARISC      Arch = "SCMP_ARCH_PARISC"
 	ArchPARISC64    Arch = "SCMP_ARCH_PARISC64"
 	ArchRISCV64     Arch = "SCMP_ARCH_RISCV64"
+	ArchLOONG64     Arch = "SCMP_ARCH_LOONGARCH64"
```

Comment:
```suggestion
	ArchLOONGARCH64     Arch = "SCMP_ARCH_LOONGARCH64"
```

Reply:
Hi [AkihiroSuda](https://github.com/AkihiroSuda), 
ArchLOONG64 is used here, because the GOARCH name corresponding to the LoongArch64 architecture in golang is loong64.
The support for loong64 will also be integrated into libseccomp-go, and the corresponding PR is: https://github.com/seccomp/libseccomp-golang/pull/106
I understand that it should be a better solution to keep the name of Arch consistent in these two projects.

Reply:
The variable name here must correspond to the seccomp arch name, not the GOARCH name.
See `ArchX86_64` vs `amd64`, `ArchAARCH64` vs `arm64`.

Reply:
> The variable name here must correspond to the seccomp arch name, not the GOARCH name. See `ArchX86_64` vs `amd64`, `ArchAARCH64` vs `arm64`.

Thanks